go/bin/lint

```diff
@@ -3,5 +3,5 @@ set -e
 
 # required by gotype to work properly
 # see https://github.com/golang/go/issues/12703#issuecomment-144429128
-go install
-gometalinter.v1 --deadline 60s ./ ./test-fixtures
+go install $(go list ./... | grep -v /vendor/)
+gometalinter.v1 --deadline 60s ./
```

Comment:
gometalinter is not recursive so you need to specify the directories you want to lint